scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,23 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    def test_nist(self):
+        # The npz file with the nist test cases data is assumed to be in the
+        # same directory as this file.
+        nist = np.load(os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                    'nist_anova.npz')))
+        rtol = 1e-7
```

Comment:
``rtol=1e-7`` needs to be inside of the for-loop, otherwise it'll remain at 1e-4 after the first time it's changed to that value